aosp.xml

```diff
@@ -101,6 +101,7 @@
   <project path="external/ipsec-tools" name="platform/external/ipsec-tools" groups="pdk" />
   <project path="external/iptables" name="platform/external/iptables" groups="pdk" />
   <project path="external/iputils" name="platform/external/iputils" groups="pdk" />
+  <project path="external/iw" name="platform/external/iw" groups="pdk" />
```

Comment:
We don't need that on device (Pixel 3a at least), so you should add it in `emulator-10.xml`